libs/blocks/merch-card/merch-card.css

```diff
@@ -0,0 +1,359 @@
+}
+
+.merch-card .consonant-SegmentBlade-description span.placeholder-resolved,
+.merch-card .consonant-SpecialOffers-description span.placeholder-resolved,
+.merch-card .consonant-PlansCard-description span.placeholder-resolved {
+  font-size: var(--type-heading-xs-size);;
+}
+
+.merch-card .consonant-SegmentBlade-description span.placeholder-resolved[data-template="priceStrikethrough"],
+.merch-card .consonant-SpecialOffers-description span.placeholder-resolved[data-template="priceStrikethrough"],
+.merch-card .consonant-PlansCard-description span.placeholder-resolved[data-template="priceStrikethrough"] {
+  font-size: var(--type-body-xs-size);
+}
+
+.merch-card.consonant-SpecialOffers-Card {
+  border-radius: 16px;
+}
+
+.merch-card .consonant-SpecialOffers-img,
+.merch-card .consonant-PlansCard-img {
+  flex-grow: 1;
+  position: relative;
+  width: 100%;
+  min-height: 213px;
+  max-height: 213px;
+  background-color: var(--background-color);
+  background-position: 50% 50%;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.merch-card .consonant-SpecialOffers-ribbon,
+.merch-card .consonant-SegmentBlade-ribbon,
+.merch-card .consonant-PlansCard-ribbon {
+  position: absolute;
+  top: 16px;
+  right: 0;
+  font-size: var(--type-heading-xxs-size);
+  font-weight: 500;
+  max-width: 150px;
+  line-height: 16px;
+  text-align: center;
+  padding: 8px 11px;
+  border-radius: 5px 0 0 5px;
+}
+
+.merch-card .consonant-ProductCard-cell,
+.merch-card .consonant-SegmentBlade-cell,
+.merch-card .consonant-SpecialOffers-cell,
+.merch-card .consonant-PlansCard-cell {
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.merch-card p {
+  font-size: var(--type-body-xs-size);
+  font-weight: 400;
+  line-height: 1.5;
+  margin-bottom: var(--spacing-xxs);
+  margin-top: 0;
+  color: var(--color-black);
+}
+
+.merch-card .consonant-SpecialOffers-inner,
+.merch-card .consonant-PlansCard-inner {
+  justify-content: center;
+}
+
+.merch-card.special-offers .consonant-SpecialOffers-description p,
+.merch-card.special-offers .consonant-PlansCard-description p {
+  font-size: var(--type-body-xs-size);
+}
+
+.merch-card.special-offers .consonant-CardFooter {
+  margin-top: 0;
+  padding-top: 0;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.merch-card.special-offers h4 {
+  font-size: var(--type-body-xxs-size);
+  text-transform: uppercase;
+  letter-spacing: 1px;
+}
+
+.merch-card .consonant-CardFooter .standard-wrapper,
+.merch-card .consonant-CardFooter .checkbox-container {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.merch-card .consonant-CardFooter .checkbox-container {
+  justify-content: flex-start;
+  margin-bottom: var(--spacing-s);
+  cursor: pointer;
+}
+
+.merch-card .consonant-CardFooter .standard-wrapper {
+  height: 32px;
+  white-space: nowrap;
+}
+
+.merch-card .secure-transaction-wrapper {
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
```

Comment:
Is this correct. I tried to add a long label, aka German, and the text got cut off. Id think this should wrap w/ appropriate line height? Have you tested these labels w/ longer lang support?